package/MDAnalysis/analysis/base.py

```diff
@@ -447,8 +479,39 @@ def run(self, start=None, stop=None, step=None, frames=None,
             self.times[i] = ts.time
             self._single_frame()
         logger.info("Finishing up")
+        return self
+
+    def _setup_bslices(self, start=None, stop=None, step=None, frames=None):
+        self._bslices = [(start, stop, step, frames)]
+    
+    def _setup_scheduler(self, **kwa):
```

Comment:
We typically write out `**kwargs` in the code.

Reply:
fixed